web/src/hooks/colors.ts

```diff
@@ -1,11 +1,11 @@
 const defaultCo2Scale = {
-  steps: [0, 150, 600, 750, 800],
-  colors: ['#2AA364', '#F5EB4D', '#9E4229', '#381D02', '#381D02'],
+  steps: [0, 150, 600, 800, 1100, 1500],
+  colors: ['#2AA364', '#F5EB4D', '#9E4229', '#381D02', '#381D02', '#000'],
 };
 
 const colorblindCo2Scale = {
-  steps: [0, 200, 400, 600, 800],
-  colors: ['#FFFFB0', '#E0B040', '#A06030', '#602020', '#000010'],
+  steps: [0, 150, 600, 800, 1100, 1500],
+  colors: ['#FFFFB0', '#E3BF66', '#BB833C', '#8B4D2B', '#4E241F', '#000000'],
```

Comment:
```suggestion
  colors: ['#FFFFB0', '#E3BF66', '#BB833C', '#8B4D2B', '#4E241F', '#000'],
```
Super tiny nitpick that vite is most likely optimizing anyway 😅